### ISSUE TITLE
[Bug]: CROP function - submit button does nothing

### ISSUE THREAD

**Issue by kriss6557** [Stirling-Tools/Stirling-PDF#2377]
### Installation Method

Docker

### The Problem

Hi,
When selecting a pdf in the crop tool, drawing a selection rectangle in red then clicking on submit does nothing

### Version of Stirling-PDF

latest, 0.35.0, 0.34.0

### Docker Configuration

```txt
docker-compose sample from the official readme
```

### Relevant Log Output

```txt
No log available
```

### Additional Information

MacOS

### Browsers Affected

Firefox, Safari

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

**Comment by qma-ibf** [comment]
Same here. The console show the following error: 
 
<img width="1455" alt="Screenshot 2024-12-03 at 10 52 05" src="https://github.com/user-attachments/assets/47c67bbd-c0fc-40c0-9801-4478deaa00e3">

**Comment by Frooodle** [comment]
fixed!

**Comment by kriss6557** [comment]
Well it might be fixed but in 0.36.X and latest I cannot upload or drap&drop any file in the Tools menu. I can upload files in the multitool menu though
